store/dbstore.go

```diff
@@ -26,7 +26,9 @@ import (
 )
 
 const (
-	DBStoreName = "mender-store"
+	DBStoreName         = "mender-store"
+	dbFileBrokenPrefix  = "-broken"
+	dbFileWorkingPrefix = "-working"
```

Comment:
As mentioned on Slack, I'd try to minimize the diff and keep the change concise. We don't need to backup the working database, so I'd remove all this part. All we need to do is to rename `mender-store` into something else (e.g. `mender-store-broken`) and create a new empty database if opening the LMDB fails. That's it, nothing else.

Reply:
fixed.